presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -492,9 +492,9 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
         }
     }
 
-    protected String createTableSql(RemoteTableName remoteTableName, List<String> columns)
+    protected String createTableSql(RemoteTableName remoteTableName, List<String> columns, String tableProperties)
```

Comment:
The overridable method should get `TableMetadata`. 
We should assume the table properties are translated into single contigyous string